jobs/BaseJob.php

```diff
@@ -47,12 +47,53 @@ abstract class BaseJob implements ShouldQueue
      */
     public $queue;
 
+    /**
+     * The number of times the job may be attempted.
+     *
+     * @var int
+     */
+    public $tries = 3;
+
+    /**
+     * The number of SECONDS to wait before retrying the job.
+     *
+     * @var int
+     */
+    public $backoff = 5;
+
+    /**
+     * The maximum number of SECONDS a job should get processed before consider failed
+     *
+     * @var int
+     */
+    public $timeout = 120;
+
+    /**
+     * The maximum number of unhandled exceptions to allow before failing.
+     *
+     * @var int
+     */
+    public $maxExceptions = 3;
+
+    /**
+     * Indicate if the job should be marked as failed on timeout.
+     *
+     * @var bool
+     */
+    public $failOnTimeout = true;
+
+    /**
+     * Initiate the job
```

Comment:
Should be `Initialize` rather than `Initiate`, I think -- `Initiate` implies that calling the constructor starts the job. (Also below)

Reply:
done .